### PR TITLE
Mongo naming guidelines

### PR DESCRIPTION
**Pull request by rhomes-wcd**
Modified the README to include Mongo naming guidelines.

### CODE REVIEW

**Comment by rhomes-wcd** [comment]
Closes #174

**Comment by sneakysneakysneaky** [comment]
Does this belong in a separate file? Merely referenced by the README?

File: README.md [hunk 1]
```diff
@@ -29,6 +29,17 @@ Coding Standards
...
 * All CSS code complies to [BEM](http://getbem.com/) standards.
+* Mongo standards
+  * Collection naming
+    * camelCase collection names to coincide with Meteor style guidelines for identifiers.
+    * Collection names should not start with special characters, such as an underscore. Doing so will introduce problems when interacting with the collection via the MongoDB shell.
+    * Make plural as a collection contains a multitude of instances of the object being described.
+    * No word separators should be used as collection names should be short and direct.
+  * Field naming
+    * camelCase field names to coincide with Meteor style guidelines for identifiers.
+    * Collection names should not start with special characters to avoid issues with the MongoDB shell.
```
Comment (jimrollenhagen) [inline README.md]: You have this one twice :P

**Comment by jimrollenhagen** [comment]
Agree, this should be in `doc/dev/` somewhere.

File: README.md (hunk 1 above)
Comment (rhomes-wcd) [inline README.md]: Gah, thanks. Will update to read as 'fields' and will move to a separate doc and reference the document in the readme after work today.

**Comment by rhomes-wcd** [comment]
This should be good to go now.

**Pull request merged**